inspirehep/modules/orcid/tasks.py

```diff
@@ -165,6 +167,12 @@ def orcid_push(self, orcid, rec_id, oauth_token):
         rec_id(int): inspire record's id to push to ORCID.
         oauth_token(string): orcid token.
     """
+    compiled = re.compile(current_app.config.get(
```

Comment:
Wouldn't it be worth it to compile it in the config so it only happens once? Also in any case I think `re.match` can do the compilation for you if you pass the string so in this instance it doesn't seem necessary, since you only pass it once

Reply:
I'd prefer to keep the config file as dumb as possible, just a set of key = values statements and possibly json serializable such that it can evolve into a configuration service (like ETCD).
And for these 2 lines, I initially wrote the code this way because shorter statements are more readable, but actually, I am going to change it.

Reply:
Changed